Havit.Blazor.Components.Web.Bootstrap/Grids/HxGrid.razor

```diff
@@ -104,6 +105,24 @@
 				</tr>
 			</thead>
 			<tbody>
+                @{
+                    RenderFragment<int> placeholderRowTemplate = (int rowIndex) =>
+                    @<tr @key="@rowIndex"
+                                height="@((ContentNavigationModeEffective == GridContentNavigationMode.InfiniteScroll) ? rowHeight : null)"
+                                class="@ItemRowCssClassEffective"
+                            @onclick:stopPropagation>
+
+                            @foreach (IHxGridColumn<TItem> column in columnsToRender)
+                            {
+                                // same as above this else section
+                                GridCellTemplate cellTemplate = column.GetItemPlaceholderCellTemplate(new GridPlaceholderCellContext { Index = rowIndex });
+                                <td @key="@column" class="@cellTemplate.CssClass">
```

Comment:
The `cellTemplate.CssClass` remains unset, we have to adjust `GetItemPlaceholderCellTemplate()` method to set the `CssClass` of `GridCellTemplate` returned.